.travis.yml

```diff
@@ -1,6 +1,18 @@
-dist: bionic
```

Comment:
I undestand that this was removed because OSX but I would still like to explicitly run it under bionic; I'm not sure how to do that though.

Reply:
We don't need any specific distro. Travis will use it's own distro then which is much more lightweight then an Ubuntu. We could use bionic if we need any other dependencies.

Reply:
Actually no, Travis uses Ubuntu for os: linux; see https://docs.travis-ci.com/user/reference/linux/. By default it uses Ubuntu 16.04 (xenial) which is outdated; I want to make sure that builds work on the latest versions of compilers and the like hence why I want to specify 18.04 (bionic).

Reply:
Oh didn't knew that!
Will update to bionic

Reply:
Fixed